projects/plugins/protect/src/js/components/firewall-page/index.jsx

```diff
@@ -161,96 +162,137 @@ const FirewallPage = () => {
 		API.wafSeen();
 	}, [ isSeen, setWafIsSeen ] );
 
+	const [ showManualRules, setShowManualRules ] = useState( false );
+
+	const handleShowManualRulesClick = useCallback( () => {
+		if ( showManualRules ) {
+			return setShowManualRules( false );
+		}
+		setShowManualRules( true );
```

Comment:
```suggestion
		setShowManualRules( ! showManualRules );
```
Current solution works great, this just one-lines it.